### PR TITLE
Add hpo ingest

### PR DESCRIPTION
**Pull request by justaddcoffee**
To address #48

### CODE REVIEW

**Comment by deepakunni3** [comment]
@justaddcoffee Is this ready for review?

**Comment by justaddcoffee** [comment]
@deepakunni3 yep review away (I promise I'll stop committing to this branch)

**Comment by justaddcoffee** [comment]
@deepakunni3 thanks! Answers a lot of questions that I had, but didn't ask. Will update PR soon

**Comment by justaddcoffee** [comment]
Thanks @deepakunni3 ! All sensible suggestions, should be fixed now, have a look

**Comment by deepakunni3** [comment]
Thanks @justaddcoffee. Looks good! 👍

File: download.yaml [hunk 1]
```diff
@@ -52,6 +52,13 @@
   local_name: gene_info.gz
 
 #
+# Human phenotype ontology (HPO) - no annotations for now
+#
+-
+  # HPO obo file
+  url: https://raw.githubusercontent.com/obophenotype/human-phenotype-ontology/master/hp.obo
```
Comment (cmungall) [inline download.yaml:59]: I recommend always using OBO PURLs

**Comment by justaddcoffee** [comment]
thanks @cmungall ! Just updated this with a nano-PR

**Pull request merged**